pkg/upgrade/container/container_test.go

```diff
@@ -0,0 +1,150 @@
+package container_test
```

Comment:
When you're settled on a test framework, would be good to add a "how to run test suite" section to docs.